tests/testthat/test-variables.R

```diff
@@ -130,7 +131,6 @@ with_test_authentication({
     test_that("before modifying", {
         expect_identical(name(ds$v1), "v1")
         expect_identical(description(ds$v2), "")
-        expect_identical(alias(ds$v1), "v1")
```

Comment:
Was removing this an accident? I'm having trouble figuring out if it's needed or not?

Reply:
Was indeed an accident, sorry!

Reply:
Weirdly I don't see this line as being removed in my local repo.... `git confused`